qc/homo_sapiens_qc.py

```diff
@@ -771,3 +771,89 @@ def __init__(self):
                 destination=8),
         ]
         self.demographic_events.sort(key=lambda x: x.time)
+
+
+class ZigZag(models.DemographicModel):
+    """
+    Model from Schiffels et al (2014) used to test inference methods on a "zigzag"
+    demography. Single population, repeated growth and decay of pop size.
+    """
+    def __init__(self):
+        # from section 7 of the supplement
+        self.generation_time = 30
+        mu = 1.25e-8
+        L = 10000000
+        theta = 7156
+        Ne = theta / 4 / mu / L
+
+        self.population_configurations = [
+            msprime.PopulationConfiguration(
+                initial_size=Ne, growth_rate=0,
+                metadata={"sampling_time": 0})
+        ]
+
+        # I checked these values against the catalog model by hand, and then switched
+        # to match exactly the values in the catalog model
+        # rates are set at various times
```

Comment:
I don't quite understand the motivation here. Did your commented out code not pass QC for some reason?

Reply:
The catalog model has sizes and times that were rounded when they were transcribed, but in my first qc approach I converted from the ms commands to times and growth rates internally. So there are small fractions of difference between the models due to math on rounded floats.

Reply:
Hmm, this shouldn't be necessary. We compare the values to within a given tolerance, which should mop up the floating point errors. Perhaps the values in the real model are just put in at too low a precision? I don't think we need to worry about blinding etc here. 

Your commented out QC model is much better, I think it would be better to update the values in the main model.